schedule/yast/modify_existing_partition_sle_aarch.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name:           modify_existing_partition@aarch
```

Comment:
aarch64 maybe better name, also in the file as well.

Reply:
fixed